### PR TITLE
Logizo - remove tests

### PR DESCRIPTION
**Pull request by ChanochShayner**
## XSOAR [Partner](https://xsoar.pan.dev/docs/partners/become-a-tech-partner)?
In case you are a technological partner:
- [ ] The title must be in the following format: **[YOUR_PARTNER_ID] short description**.

If you are not a technological partner (i.e. PANW employee, customer or individual developer), please delete this section.

## Status
- [] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/26729#

## Description
remove the test from conf.json and add - No tests to the integration yml.
## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by ShahafBenYakir — APPROVED** [review]

**Review by guykeller — APPROVED** [review]

File: Packs/Logzio/ReleaseNotes/1_1_1.md
Comment (ShahafBenYakir) [inline Packs/Logzio/ReleaseNotes/1_1_1.md:4]:
```suggestion
- Change the docker image to - 3.8.5.10845.
```

**Comment by ShahafBenYakir** [comment]
Why have you moved the test PB to the NonCircle folder?

**Review by ShahafBenYakir — CHANGES_REQUESTED** [review]
Remove the NonCircleFolder

**Comment by Itay4** [comment]
@ShahafBenYakir @ChanochShayner what is the status here?

**Comment by ShahafBenYakir** [comment]
@Itay4  There is a bug in the flow of collecting tests we have a meeting on this tomorrow which after we'll decide what is needed.

**Comment by ShahafBenYakir** [comment]
@ChanochShayner The issue was fixed pull and re run

**Comment by ChanochShayner** [comment]
@ShahafBenYakir I'm not sure I understand, what's supposed to change in this PR?
At the moment the changes are - removing the tests from logzio, is that what it should be?

**Comment by ShahafBenYakir** [comment]
@ChanochShayner Yes. Your build failed because of an issue with the collect test that was fxied.

**Comment by ChanochShayner** [comment]
@ShahafBenYakir Waiting for your approval, thanks.

**Review by ShahafBenYakir — APPROVED** [review]

**Pull request merged**
